android/src/main/java/com/revenuecat/purchases/hybridcommon/common.kt

```diff
@@ -261,12 +261,13 @@ fun invalidatePurchaserInfoCache() {
 }
 
 fun canMakePayments(context: Context,
-                    features: List<String>,
+                    features: List<Int>,
                     onResult: OnResultAny<Boolean>) {
     val billingFeatures = mutableListOf<BillingFeature>()
     try {
-        billingFeatures.addAll(features.map { BillingFeature.valueOf(it) })
-    } catch (e: IllegalArgumentException) {
+        val billingFeatureEnumValues = BillingFeature.values()
+        billingFeatures.addAll(features.map { billingFeatureEnumValues[it] })
```

Comment:
@aboedo Here's that refactor we talked about. When I make the update to `purchases-android` to include a store parameter, we could consider updating the BillingFeature enum there to take a `code` the way we did for `PurchasesError`. I'm not sure that's necessary though